### ISSUE TITLE
uMatrix installs itself even if I haven't checked the box to do so

### ISSUE THREAD

**Issue by ghost** [allo-/firefox-profilemaker#233]
i just made a new profile to get rid of this persisting ~~garbage~~ of an extension and to my surprise, it reinstalled itself, though I have deleted everything on my profile folder + I haven't checked it on the ffprofile website

**Comment by allo-** [comment]
Duplicate of #232

**Comment by allo-** [comment]
I can reproduce the bug when using enterprise policies.

Other addons pull in unselected addons as well. For example `Cookie Autodelete` only creates the policy:

```
{
  "policies": {
    "ExtensionSettings": {
      "CookieAutoDelete@kennydo.com": {
        "install_url": "https://addons.mozilla.org/firefox/downloads/latest/cookie-autodelete/latest.xpi",
        "installation_mode": "normal_installed"
      },
      "uBlock0@raymondhill.net": {
        "install_url": "https://addons.mozilla.org/firefox/downloads/latest/ublock-origin/latest.xpi",
        "installation_mode": "normal_installed"
      },
      "jid1-MnnxcxisBPnSXQ@jetpack": {
        "install_url": "https://addons.mozilla.org/firefox/downloads/latest/privacy-badger17/latest.xpi",
        "installation_mode": "normal_installed"
      },
      "https-everywhere@eff.org": {
        "install_url": "https://addons.mozilla.org/firefox/downloads/latest/https-everywhere/latest.xpi",
        "installation_mode": "normal_installed"
      },
      "CanvasBlocker@kkapsner.net": {
        "install_url": "https://addons.mozilla.org/firefox/downloads/latest/canvasblocker/latest.xpi",
        "installation_mode": "normal_installed"
      }
    }
  }
}
```

I think we had another bug with enterprise policies, which is still unresolved #220. I had disabled the enterprise policy download button as workaround on the production site, but it got activated after some update again.

**Comment by allo-** [comment]
@km-ch I believe this may be fixed by the latest commit. Can you test again?

**Comment by allo-** [comment]
Closed by #272.
